### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-tf-module.git`](https://github.com/cisagov/skeleton-tf-module.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/ipsec-tunnel-tf-module.git ipsec-tunnel-tf-module
    cd ipsec-tunnel-tf-module
    git remote add skeleton https://github.com/cisagov/skeleton-tf-module.git
    git remote set-url --push skeleton no_push
    git switch develop
    git checkout -b lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the
   branch, `commit`, and `push` your changes:

    ```console
    git add .github/CODEOWNERS README.md examples/basic_usage/README.md examples/basic_usage/main.tf examples/basic_usage/outputs.tf examples/basic_usage/variables.tf main.tf outputs.tf variables.tf 
    git commit
    git push --force --set-upstream origin lineage/skeleton
    ```

    Note that you may _append_ to the default merge commit message
    that git creates for you, but *please do not delete the existing
    content*.  It provides useful information about the merge that is
    being performed.

1. Wait for all the automated tests to pass.

1. Check the "Everything is cool" checkbox below:

    - [x] ✌️ The conflicts in this pull request have been resolved.

1. Mark this draft pull request "Ready for review".

------------

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]

**Review by dav3r — APPROVED** [review]
🥨

File: README.md [hunk 1]
```diff
@@ -33,33 +33,47 @@ module "ipsec_tunnel" {
...
-| remote_cidr_blocks | A list of CIDR blocks associated with the remote (non-AWS) end of the tunnel (e.g. [192.168.11.0/24, 10.10.11.0/24]) | `list(string)` | n/a | yes |
-| remote_ip | The public IP of the remote (non-AWS) end of the tunnel (e.g. 1.2.3.4) | `string` | n/a | yes |
-| route_table_ids | A list of ID corresponding to route tables that should route traffic through the VPN/IPSec gateway (e.g. [rtb-0123456789abcdef0, rtb-0123456789abcdef1]) | `list(string)` | n/a | yes |
-| static_routes_only | A boolean value indicating whether the VPN connection uses static routes exclusively.  Static routes must be used for devices that don't support BGP. | `bool` | `true` | no |
-| tags | Tags to apply to all AWS resources created | `map(string)` | `{}` | no |
-| vpc_id | The ID of the AWS VPC where the Virtual private Gateway is to be deployed (e.g. vpc-0123456789abcdef0) | `string` | n/a | yes |
+| aws\_availability\_zone | The AWS availability zone where the Virtual Private Gateway is to be deployed (e.g. a, b, c, etc.) | `string` | n/a | yes |
```
Comment (mcdonnnj) [inline README.md]:
```suggestion
| aws\_availability\_zone | The AWS availability zone where the Virtual Private Gateway is to be deployed (e.g. a, b, c, etc.). | `string` | n/a | yes |
```

File: README.md [hunk 2]
```diff
@@ -33,33 +33,47 @@ module "ipsec_tunnel" {
...
+| aws\_region | The AWS region where the Virtual Private Gateway is to be deployed (e.g. us-east-1) | `string` | n/a | yes |
```
Comment (mcdonnnj) [inline README.md]:
```suggestion
| aws\_region | The AWS region where the Virtual Private Gateway is to be deployed (e.g. "us-east-1"). | `string` | n/a | yes |
```

File: README.md [hunk 3]
```diff
@@ -33,33 +33,47 @@ module "ipsec_tunnel" {
...
+| remote\_bgp\_asn | The BGP ASN of the remote (non-AWS) end of the tunnel (e.g. 11111).  This value is not used if static\_routes\_only is true. | `number` | `65000` | no |
+| remote\_cidr\_blocks | A list of CIDR blocks associated with the remote (non-AWS) end of the tunnel (e.g. [192.168.11.0/24, 10.10.11.0/24]) | `list(string)` | n/a | yes |
```
Comment (mcdonnnj) [inline README.md]:
```suggestion
| remote\_cidr\_blocks | A list of CIDR blocks associated with the remote (non-AWS) end of the tunnel (e.g. [192.168.11.0/24, 10.10.11.0/24]). | `list(string)` | n/a | yes |
```

File: README.md [hunk 4]
```diff
@@ -33,33 +33,47 @@ module "ipsec_tunnel" {
...
+| remote\_ip | The public IP of the remote (non-AWS) end of the tunnel (e.g. 1.2.3.4) | `string` | n/a | yes |
```
Comment (mcdonnnj) [inline README.md]:
```suggestion
| remote\_ip | The public IP of the remote (non-AWS) end of the tunnel (e.g. 1.2.3.4). | `string` | n/a | yes |
```

File: README.md [hunk 5]
```diff
@@ -33,33 +33,47 @@ module "ipsec_tunnel" {
...
+| route\_table\_ids | A list of ID corresponding to route tables that should route traffic through the VPN/IPSec gateway (e.g. [rtb-0123456789abcdef0, rtb-0123456789abcdef1]) | `list(string)` | n/a | yes |
```
Comment (mcdonnnj) [inline README.md]:
```suggestion
| route\_table\_ids | A list of ID corresponding to route tables that should route traffic through the VPN/IPSec gateway (e.g. [rtb-0123456789abcdef0, rtb-0123456789abcdef1]). | `list(string)` | n/a | yes |
```

File: README.md [hunk 6]
```diff
@@ -33,33 +33,47 @@ module "ipsec_tunnel" {
...
+| static\_routes\_only | A boolean value indicating whether the VPN connection uses static routes exclusively.  Static routes must be used for devices that don't support BGP. | `bool` | `true` | no |
+| vpc\_id | The ID of the AWS VPC where the Virtual private Gateway is to be deployed (e.g. vpc-0123456789abcdef0) | `string` | n/a | yes |
```
Comment (mcdonnnj) [inline README.md]:
```suggestion
| vpc\_id | The ID of the AWS VPC where the Virtual private Gateway is to be deployed (e.g. vpc-0123456789abcdef0). | `string` | n/a | yes |
```

File: examples/basic_usage/README.md [hunk 7]
```diff
@@ -18,20 +18,36 @@ No requirements.
...
 ## Inputs ##
 
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:--------:|
-| aws_availability_zone | The AWS availability zone where the Virtual Private Gateway is to be deployed (e.g. a, b, c, etc.) | `string` | n/a | yes |
-| aws_region | The AWS region where the Virtual Private Gateway is to be deployed (e.g. us-east-1) | `string` | n/a | yes |
-| remote_cidr_blocks | A list of CIDR blocks associated with the remote (non-AWS) end of the tunnel (e.g. [192.168.11.0/24, 10.10.11.0/24]) | `list(string)` | n/a | yes |
-| remote_ip | The public IP of the remote (non-AWS) end of the tunnel (e.g. 1.2.3.4) | `string` | n/a | yes |
-| route_table_ids | A list of ID corresponding to route tables that should route traffic through the VPN/IPSec gateway (e.g. [rtb-0123456789abcdef0, rtb-0123456789abcdef1]) | `list(string)` | n/a | yes |
+| aws\_availability\_zone | The AWS availability zone where the Virtual Private Gateway is to be deployed (e.g. a, b, c, etc.) | `string` | n/a | yes |
```
Comment (mcdonnnj) [inline examples/basic_usage/README.md]:
```suggestion
| aws\_availability\_zone | The AWS availability zone where the Virtual Private Gateway is to be deployed (e.g. a, b, c, etc.). | `string` | n/a | yes |
```

File: examples/basic_usage/README.md [hunk 8]
```diff
@@ -18,20 +18,36 @@ No requirements.
...
+| aws\_region | The AWS region where the Virtual Private Gateway is to be deployed (e.g. us-east-1) | `string` | n/a | yes |
```
Comment (mcdonnnj) [inline examples/basic_usage/README.md]:
```suggestion
| aws\_region | The AWS region where the Virtual Private Gateway is to be deployed (e.g. "us-east-1"). | `string` | n/a | yes |
```

File: examples/basic_usage/README.md [hunk 9]
```diff
@@ -18,20 +18,36 @@ No requirements.
...
+| remote\_cidr\_blocks | A list of CIDR blocks associated with the remote (non-AWS) end of the tunnel (e.g. [192.168.11.0/24, 10.10.11.0/24]) | `list(string)` | n/a | yes |
```
Comment (mcdonnnj) [inline examples/basic_usage/README.md]:
```suggestion
| remote\_cidr\_blocks | A list of CIDR blocks associated with the remote (non-AWS) end of the tunnel (e.g. [192.168.11.0/24, 10.10.11.0/24]). | `list(string)` | n/a | yes |
```

File: examples/basic_usage/README.md [hunk 10]
```diff
@@ -18,20 +18,36 @@ No requirements.
...
+| remote\_ip | The public IP of the remote (non-AWS) end of the tunnel (e.g. 1.2.3.4) | `string` | n/a | yes |
```
Comment (mcdonnnj) [inline examples/basic_usage/README.md]:
```suggestion
| remote\_ip | The public IP of the remote (non-AWS) end of the tunnel (e.g. 1.2.3.4). | `string` | n/a | yes |
```

File: examples/basic_usage/README.md [hunk 11]
```diff
@@ -18,20 +18,36 @@ No requirements.
...
+| route\_table\_ids | A list of ID corresponding to route tables that should route traffic through the VPN/IPSec gateway (e.g. [rtb-0123456789abcdef0, rtb-0123456789abcdef1]) | `list(string)` | n/a | yes |
```
Comment (mcdonnnj) [inline examples/basic_usage/README.md]:
```suggestion
| route\_table\_ids | A list of ID corresponding to route tables that should route traffic through the VPN/IPSec gateway (e.g. [rtb-0123456789abcdef0, rtb-0123456789abcdef1]). | `list(string)` | n/a | yes |
```

File: examples/basic_usage/README.md [hunk 12]
```diff
@@ -18,20 +18,36 @@ No requirements.
...
 | tags | Tags to apply to all AWS resources created | `map(string)` | `{}` | no |
```
Comment (mcdonnnj) [inline examples/basic_usage/README.md]:
```suggestion
| tags | Tags to apply to all AWS resources created. | `map(string)` | `{}` | no |
```

File: examples/basic_usage/README.md [hunk 13]
```diff
@@ -18,20 +18,36 @@ No requirements.
...
-| vpc_id | The ID of the AWS VPC where the Virtual private Gateway is to be deployed (e.g. vpc-0123456789abcdef0) | `string` | n/a | yes |
+| vpc\_id | The ID of the AWS VPC where the Virtual private Gateway is to be deployed (e.g. vpc-0123456789abcdef0) | `string` | n/a | yes |
```
Comment (mcdonnnj) [inline examples/basic_usage/README.md]:
```suggestion
| vpc\_id | The ID of the AWS VPC where the Virtual private Gateway is to be deployed (e.g. vpc-0123456789abcdef0). | `string` | n/a | yes |
```

**Review by mcdonnnj — APPROVED** [review]
I leave a number of suggestions regarding periods for your consideration. The variable descriptions would also need to be updated.

File: examples/basic_usage/README.md (hunk 13 above)
Comment (jsf9k) [reply examples/basic_usage/README.md]: See commit 1c13739.

File: examples/basic_usage/README.md (hunk 12 above)
Comment (jsf9k) [reply examples/basic_usage/README.md]: See commit 1c13739.

File: examples/basic_usage/README.md (hunk 11 above)
Comment (jsf9k) [reply examples/basic_usage/README.md]: See commit 1c13739.

File: examples/basic_usage/README.md (hunk 10 above)
Comment (jsf9k) [reply examples/basic_usage/README.md]: See commit 1c13739.

File: examples/basic_usage/README.md (hunk 9 above)
Comment (jsf9k) [reply examples/basic_usage/README.md]: See commit 1c13739.

File: examples/basic_usage/README.md (hunk 8 above)
Comment (jsf9k) [reply examples/basic_usage/README.md]: See commit 1c13739.

File: examples/basic_usage/README.md (hunk 7 above)
Comment (jsf9k) [reply examples/basic_usage/README.md]: See commit 1c13739.

File: README.md (hunk 6 above)
Comment (jsf9k) [reply README.md]: See commit 1c13739.

File: README.md (hunk 5 above)
Comment (jsf9k) [reply README.md]: See commit 1c13739.

File: README.md (hunk 4 above)
Comment (jsf9k) [reply README.md]: See commit 1c13739.

File: README.md (hunk 3 above)
Comment (jsf9k) [reply README.md]: See commit 1c13739.

File: README.md (hunk 2 above)
Comment (jsf9k) [reply README.md]: See commit 1c13739.

File: README.md (hunk 1 above)
Comment (jsf9k) [reply README.md]: See commit 1c13739.

**Pull request merged**